cms/djangoapps/contentstore/toggles.py

```diff
@@ -62,3 +62,20 @@ def bypass_olx_failure_enabled():
     Check if bypass is enabled for course olx validation errors.
     """
     return BYPASS_OLX_FAILURE.is_enabled()
+
+
+# .. toggle_name: FEATURES['ENABLE_EXAM_SETTINGS_HTML_VIEW']
+# .. toggle_implementation: SettingDictToggle
+# .. toggle_default: False
+# .. toggle_description: When enabled, users can access new course authoring view
+# .. toggle_warnings: None
+# .. toggle_creation_date: 2021-06-07
```

Comment:
the date toggle_creation_date for `ENABLE_EXAM_SETTINGS_HTML_VIEW` is 
https://github.com/edx/edx-internal/blame/master/ansible/vars/stage-edx.yml#L216 23rd July 2020.